### PR TITLE
Make bucket command

### PR DESCRIPTION
**Pull request by dvaldivia**
Introduces the command `./m3 tenant mb TENANT_NAME BUCKET_NAME` and creates the tenant using the MinIO operator credentials

### CODE REVIEW

File: cluster/storage-cluster.go [hunk 1]
```diff
@@ -304,3 +304,71 @@ func createTenantInStorageGroup(ctx *Context, tenant *Tenant, sg *StorageGroup)
...
 	return ch
 }
+
+// Returns a list of tenants that are allocated to the provided `StorageGroup`
+func GetTenantStorageGroupByShortName(ctx *Context, tenantShortName string) chan *StorageGroupTenantResult {
+	ch := make(chan *StorageGroupTenantResult)
+	go func() {
+		defer close(ch)
+		if tenantShortName == "" {
+			ch <- &StorageGroupTenantResult{Error: errors.New("empty tenant short name")}
```
Comment (krisis) [inline cluster/storage-cluster.go:314]: we should return immediately after sending the error on the channel.
```suggestion
			ch <- &StorageGroupTenantResult{Error: errors.New("empty tenant short name")}
			return
```

File: cluster/storage-cluster.go [hunk 2]
```diff
@@ -304,3 +304,72 @@ func createTenantInStorageGroup(ctx *Context, tenant *Tenant, sg *StorageGroup)
...
+			FROM 
+			     m3.provisioning.tenants_storage_groups t1
+			LEFT JOIN m3.provisioning.tenants t2
+			ON t1.tenant_id = t2.id
+			LEFT JOIN m3.provisioning.storage_groups t3
+			ON t1.storage_group_id = t3.id
+			WHERE t2.short_name=$1 LIMIT 1`
+		rows, err := ctx.Tx.Query(query, tenantShortName)
+		if err != nil {
+			ch <- &StorageGroupTenantResult{Error: err}
```
Comment (krisis) [inline cluster/storage-cluster.go:329]:
```suggestion
			ch <- &StorageGroupTenantResult{Error: err}
			return
```

File: cluster/storage-cluster.go [hunk 3]
```diff
@@ -304,3 +304,72 @@ func createTenantInStorageGroup(ctx *Context, tenant *Tenant, sg *StorageGroup)
...
+		}
+		foundSomething := rows.Next()
+		if !foundSomething {
+			ch <- &StorageGroupTenantResult{Error: errors.New("tenant not found")}
```
Comment (krisis) [inline cluster/storage-cluster.go:334]:
```suggestion
			ch <- &StorageGroupTenantResult{Error: errors.New("tenant not found")}
			return
```

File: cluster/storage-cluster.go [hunk 4]
```diff
@@ -304,3 +304,72 @@ func createTenantInStorageGroup(ctx *Context, tenant *Tenant, sg *StorageGroup)
...
+			&tenantID,
+			&port,
+			&serviceName,
+			&tenantName,
+			&tenantShortName,
+			&storageGroupID,
+			&sgName,
+			&sgNum)
+		if err != nil {
+			fmt.Println(err)
```
Comment (krisis) [inline cluster/storage-cluster.go]: we should send the error on channel and return from the function.

**Review by cesnietor — APPROVED** [review]

**Pull request merged**